### PR TITLE
pl-order-blocks: CSS and HTML cleanup

### PR DESCRIPTION
**Pull request by SethPoulsen**

### CODE REVIEW

**Comment by SethPoulsen** [comment]
I know there is definitely more we could do here, but this is a good start. 

I also made it so that the headers are no longer lists, that should help with Accessibility (of course, I still need to actually make the blocks themselves keyboard accessible!)

File: elements/pl-order-blocks/pl-order-blocks.mustache
Comment (SethPoulsen) [inline elements/pl-order-blocks/pl-order-blocks.mustache:19]: Without this hidden button here, the headers of the source and target block dropzones are different sizes, which looks funny. But I'm sure there is a better way to do this than having a hidden button. Maybe we actually want to have the help button on both the source and target dropzones? When they are arranged vertically, it looks fine, but it looks a little odd when they are arranged horizontally.

File: elements/pl-order-blocks/pl-order-blocks.mustache
Comment (SethPoulsen) [inline elements/pl-order-blocks/pl-order-blocks.mustache:28]: I know its not ideal to use bootstrap's list colors (e.g. `list-group-item-primary`) for what are supposed to be headers, but its the middle of the semester so I don't feel like we should make a huge change to the existing color scheme at the moment.

File: elements/pl-order-blocks/pl-order-blocks.mustache
Comment (SethPoulsen) [inline elements/pl-order-blocks/pl-order-blocks.mustache:54]: add some spacing that should have been there from the beginning

File: elements/pl-order-blocks/pl-order-blocks.css
Comment (nwalters512) [inline elements/pl-order-blocks/pl-order-blocks.css:38]: It's a big red flag for these classes to not be scoped to this element, as they'd impact everything on the page. Can you change this to be scoped to an element-specific class like the other selectors? Ditto with everything else in this file.

File: elements/pl-order-blocks/pl-order-blocks.mustache
Comment (nwalters512) [reply elements/pl-order-blocks/pl-order-blocks.mustache:19]: If you want them to be the same height, why not just set a specific height on the container, e.g. `height: 3rem`?

I agree it would look strange to have the same popover on both panels.

**Comment by nwalters512** [comment]
I'm not sure if this is related to this PR or not, but the example question refers specifically to a yellow panel that does not seem to exist anymore:

> Drag all the prime numbers below to the yellow box.

File: elements/pl-order-blocks/pl-order-blocks.mustache
Comment (nwalters512) [reply elements/pl-order-blocks/pl-order-blocks.mustache:28]: This seems like a reasonable tradeoff.

**Comment by SethPoulsen** [comment]
> refers specifically to a yellow panel

definitely not related, but definitely still should be fixed

File: elements/pl-order-blocks/pl-order-blocks.css
Comment (SethPoulsen) [inline elements/pl-order-blocks/pl-order-blocks.css:47]: Its time to get rid of this 'inline' option altogether. I'm almost positive that no one is using it because (1) its not documented anywhere, and (2) its been halfway broken for as long as I can remember and no one has complained. 

And actually, removing it won't completely break any questions that were using it, they will just be formatted slightly differently.

**Comment by SethPoulsen** [comment]
@nwalters512 feel free to have a look at this again when you have a chance. I know you had some opinions about the borders before, I tried to make some a little bit less harsh, but happy to tweak them more if you like.

File: elements/pl-order-blocks/pl-order-blocks.mustache
Comment (nwalters512) [inline elements/pl-order-blocks/pl-order-blocks.mustache:13]: Nit: we generally err towards using Bootstrap classes instead of inline styles. e.g.:

```suggestion
    <div class="d-flex flex-wrap">
```

It doesn't make much difference in practice, but it's a convention nonetheless 🙂

File: elements/pl-order-blocks/pl-order-blocks.mustache
Comment (nwalters512) [inline elements/pl-order-blocks/pl-order-blocks.mustache:24]: Nit:

```suggestion
                <span class="mr-2">{{solution-header}}</span>
```

**Review by nwalters512 — APPROVED** [review]
LGTM! I'll leave it up to you if you want to accept the two suggestions.

**Pull request merged**
